ios/brave-ios/Sources/Brave/Frontend/Browser/Tab.swift

```diff
@@ -186,6 +187,10 @@ class Tab: NSObject {
     _syncTab
   }
 
+  private weak var braveUserAgentService: BraveUserAgentService? {
+    _braveUserAgentService
+  }
+
```

Comment:
`BraveUserAgentService` doesn't use mojom so it doesn't need to be created/destroyed the same way as the rest